client/src/app/containers/Home/active_game_list.jsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import { useSelector } from 'react-redux';
+import { useDispatch } from 'react-redux';
+import { useInjectReducer, useInjectSaga } from 'redux-injectors';
+import { selectCurrentUser, selectActiveGames } from '../../../auth/selectors';
+import { authSaga } from '../../../auth/saga';
+import { sliceKey, reducer, actions } from '../../../auth/slice';
+
+const ActiveGames = () => {
+  useInjectReducer({ key: sliceKey, reducer: reducer });
+  useInjectSaga({ key: sliceKey, saga: authSaga });
+  const dispatch = useDispatch();
+  const currentUser = useSelector(selectCurrentUser);
+  const activeGames = useSelector(selectActiveGames);
+  
+  // event handlers
+  const onRefreshActiveGames = () => {
+    dispatch(actions.refreshActiveGames(currentUser.id));
+  };
+
+  let gameButtons = [];
+  if ( activeGames ) {
+    for (const [gID, desc] of Object.entries(activeGames)) {
```

Comment:
It's more common to use an array's `map` method to render things iteratively

Reply:
And to render everything in the `return` instead of assigning a separate var

Reply:
🤔  I have been following the example here: https://stackoverflow.com/a/54550693

Perhaps you could share a code sample of what you mean?

Reply:
Oh, wait, my bad... is `activeGames` an array or a map? If it were an array (which I was assuming, but maybe not), I'd do this:

```jsx
return (
  activeGames.map(g => (
    <React.Fragment>
      <br></br>
      <button key={g.id} onClick={() => goToGame(g.id)}>
        {g.desc}
      </button>
    </React.Fragment>
  ))
)
```

If it's an object (map), you could do something similar, but I'm not sure if it's better than what you have here already:

```jsx
return (
  Object.key(activeGames).map(k => (
    <React.Fragment>
      <br></br>
      <button key={activeGames[k].id} onClick={() => goToGame(activeGames[k].id)}>
        {activeGames[k].desc}
      </button>
    </React.Fragment>
  ))
)
```